storage/scylladb_permanode.c

```diff
@@ -0,0 +1,666 @@
+/*
+ * Copyright (C) 2019-2010 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#include "scylladb_permanode.h"
+#include "common/trinary/trit_long.h"
+#define logger_id scylladb_logger_id
```

Comment:
What is `logger_id` for?
It is not used in this file.

Reply:
All files in ScyllaDB module use the same logger id.
You can reference `scylladb_utils`.

Reply:
You are right.